src/Files.App/Commands/HotKeys/IHotKeyManager.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Files.App.DataModels;
+
+namespace Files.App.Commands
+{
+	public interface IHotKeyManager
```

Comment:
It is a service that is used in several places. For example, ColumnViewBase uses it to trigger the command associated with Ctrl+A, because this key is intercepted by default by list components.

This service will be used to modify keyboard shortcuts (it is not yet used nor persistent) and will therefore be used by the view to modify them.

Reply:
Could you rename it to a Service and move it to appropriate folder?